rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

```diff
@@ -4,7 +4,7 @@ The third phase of process automation is the *Test* phase. In this phase, you ca
 
 When the tests are successful, transition to *Production* phase.
 
-The final phase in the life-cycle of a process is the *Production* phase. This phase is structured similarly to the *Test* phase. The difference is that you do not have to start the individual process runs manually. Instead you schedule a process run configuration which is then executed automatically at defined times. Or you create an invokable run configuration and publish it to Anypoint Exchange.
+The final phase in the life-cycle of a process is the *Production* phase. This phase is structured similarly to the *Test* phase. The difference is that you do not have to start the individual process runs manually. Instead you schedule a process run configuration which is then executed automatically at defined times. Or you create an invocable run configuration and publish it to Anypoint Exchange.
```

Comment:
Do you have to bold the phase names?

Reply:
[General edits for better usability and consistency](https://gus.lightning.force.com/a07EE00001peMotYAE)